core/src/main/java/org/apache/accumulo/core/iteratorsImpl/system/InterruptibleIterator.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.accumulo.core.iterators.system;
+package org.apache.accumulo.core.iteratorsImpl.system;
```

Comment:
> Wasn't the InterruptibleIterator created as an interface for users? 

This should only be used internally.  Once we get APILyzer running on the iter packages hopefully it would catch any use.  Also, it seems like any use would show up in the diff for this patch.

Reply:
Looking through the diffs I did see the SortedMapIterator class ref  InterruptibleIterator

Reply:
Looking at the other classes implementing InterruptibleIterator, I agree this should be internal use only.